src/coreclr/gc/gc.cpp

```diff
@@ -12227,6 +12081,7 @@ gc_heap::init_gc_heap (int  h_number)
     verify_pinned_queue_p = FALSE;
 #endif // _DEBUG && VERIFY_HEAP
 
+#ifdef FEATURE_LOH_COMPACTION
```

Comment:
I wonder if we should consider FEATURE_LOH_COMPACTION to be always on, like PARALLEL_MARK_LIST_SORT?

Reply:
you are right; we should. I will do that.

Reply:
actually thinking about it a bit more - there's benefit to have it because it works exclusively with custom alignment (FEATURE_STRUCTALIGN). so I will leave it for now (we could get rid of both when we make the custom alignment feature work, if we so choose to).